frontend/pages/organizations/[organizationUrl].js

```diff
@@ -112,7 +112,7 @@ export default function OrganizationPage({
           organization={organization}
           projects={projects}
           members={members}
-          organizationTypes={organizationTypes}
+          organizationTypes={organizationTypes} // not sure this does anything (not used in organization layout props)
```

Comment:
What happens if you remove it? Are the organization types still shown then?

Reply:
The function that is called "getOrganizationTypes" simply returns [] and is not the way types are returned. Types are obtained via API call listing the organization via its url slug. Furthermore we see it is passed as a prop to OrganizationLayout, but OrganizatonLayout does not have it as a parameter, so I'd suggest removing it.